### PR TITLE
fix #21098

### LINKED ISSUE #21098: There is a limitation of number of words "in" operator.
I got feedback from my customer that there is a limitation number when using "in" operator in $filter.
However, there is no reference in our official document.
If I use advanced filter (ConsistencyLevel: eventual), I could deal with the error, could use more than 15 words in "in" operator. We should explain the limitation and how to deal with it.

Example query:
https://graph.microsoft.com/v1.0/users?$count=true&$filter=userPrincipalName in ('aaa@m365x61971868.onmicrosoft.com','aaa2@m365x61971868.onmicrosoft.com','aaa3@m365x61971868.onmicrosoft.com','aaa4@m365x61971868.onmicrosoft.com','aaa5@m365x61971868.onmicrosoft.com','aaa6@m365x61971868.onmicrosoft.com','aaa7@m365x61971868.onmicrosoft.com','aaa8@m365x61971868.onmicrosoft.com','aaa9@m365x61971868.onmicrosoft.com','aaa10@m365x61971868.onmicrosoft.com','aaa11@m365x61971868.onmicrosoft.com','aaa12@m365x61971868.onmicrosoft.com','aaa13@m365x61971868.onmicrosoft.com','aaa14@m365x61971868.onmicrosoft.com','aaa15@m365x61971868.onmicrosoft.com','aaa16@m365x61971868.onmicrosoft.com')   

Error:
Too many child clauses specified in search filter expression containing 'OR' operators: 16. Max allowed: 15

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ca630f7c-e88d-3b10-4ed1-3574dd415ad2
* Version Independent ID: ca630f7c-e88d-3b10-4ed1-3574dd415ad2
* Content: [Use the filter query parameter to filter a collection of objects - Microsoft Graph](https://learn.microsoft.com/en-us/graph/filter-query-parameter?tabs=http)
* Content Source: [concepts/filter-query-parameter.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/filter-query-parameter.md)
* Product: **applications**
* Technology: **microsoft-graph**
* GitHub Login: @FaithOmbongi
* Microsoft Alias: **ombongifaith**

### PR DESCRIPTION
**Pull request by FaithOmbongi**
fix #21098

### CODE REVIEW

File: concepts/filter-query-parameter.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Support for `$filter` operators varies across Microsoft Graph APIs. The followin
 
 | Operator type         | Operator                                                                                                                                         |
 | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Equality operators    | <ul><li> Equals (`eq`) </li><li> Not equals (`ne`)</li><li> Logical negation (`not`)</li><li> In (`in`)</li><li> Has (`has`)</li></ul>                                |
+| Equality operators    | <ul><li> Equals (`eq`) </li><li> Not equals (`ne`)</li><li> Logical negation (`not`)</li><li> In (`in`)</li><li> Has (`has`)</li></ul> <br/><br/> The maximum number of expressions in a filter clause when using the `in` operator is 15 items by default and 2048 characters when using [advanced queries](./aad-advanced-queries.md).                               |
```
Comment (Licantrop0) [inline concepts/filter-query-parameter.md:27]: 2048 is the limit of the *total url* length (including the base URL and other query parameters), not just for the text inside the `in` operator.

**Review by Licantrop0 — CHANGES_REQUESTED** [review]
need to be specific of what '2048' characters we refer to.

File: concepts/filter-query-parameter.md (hunk 1 above)
Comment (FaithOmbongi) [reply concepts/filter-query-parameter.md]: Applied this feedback.

File: concepts/filter-query-parameter.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Support for `$filter` operators varies across Microsoft Graph APIs. The followin
 
 | Operator type         | Operator                                                                                                                                         |
 | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Equality operators    | <ul><li> Equals (`eq`) </li><li> Not equals (`ne`)</li><li> Logical negation (`not`)</li><li> In (`in`)</li><li> Has (`has`)</li></ul>                                |
+| Equality operators    | <ul><li> Equals (`eq`) </li><li> Not equals (`ne`)</li><li> Logical negation (`not`)</li><li> In (`in`)</li><li> Has (`has`)</li></ul> <br/><br/> **Note:**When using the `in` operator, the request is limited to 15 expressions in the filter clause by default or a URL length of 2,048 characters when using [advanced query capabilities](./aad-advanced-queries.md).                               |
```
Comment (Licantrop0) [inline concepts/filter-query-parameter.md:27]: there should be a space between **Note:** and the next word.

**Review by Licantrop0 — APPROVED** [review]

**Pull request merged**